sn_node/src/node.rs

```diff
@@ -759,9 +759,8 @@ impl Node {
         }
     }
 
-    // Shall be disabled after Beta
-    #[cfg(feature = "reward-forward")]
```

Comment:
feature guard is required, unless the entire feature gate is removed ?

Reply:
I think we want this feature in general, but I didn't want to go into the extra config here to remove the feat entirely (as forwarding must be optional in the end)